### DISCUSSION TITLE
Modbus Error

### DISCUSSION THREAD

**Discussion by colincreed** [artisan-roaster-scope/artisan#1218] [General]
As a new user, I downloaded the app and keep getting a modbus error, failed to connect. I am using windows 11. Any recommendations on rectifying
 this?

**Comment by MAKOMO** [comment]
MODBUS is a complex protocol. Are you using MODBUS RTU, ASCI, Binary, TCP, UDP? What do you try to connect to? How did you physically connect your PC to that thing? How did you configure the Artisan MODBUS side? What is your exact Artisan configuration?
